ui/slider.js

```diff
@@ -157,6 +158,48 @@ return $.widget( "ui.slider", $.ui.mouse, {
 		}
 	},
 
+	_cleanValues: function( index, newValue ) {
+		var i = 0, values ;
+
+		if ( arguments.length > 1 ) {
+			values = this.options.values;
+			if ( index === 0 && newValue < this._valueMin() ){
+				newValue = this._valueMin();
+			}
+			if ( index === 1 && newValue > this._valueMax() ){
+				newValue = this._valueMax();
+			}
+			if ( index === 0 && newValue > values[ index + 1 ] ) {
+				return values[ index + 1 ];
+			}
+			if ( index === 1 && newValue < values[ index - 1 ] ) {
+				return values[ index - 1 ];
+			}
+			return newValue;
+		}
+
+		if ( arguments.length ) {
```

Comment:
Am I missing something here? There are multiple calls to this method with no parameters, but we do nothing in that case?